axon-framework/modules.md

```diff
@@ -153,3 +155,69 @@ This module provides support for distributed tracing of Axon applications. The [
 ### Axon Tracing Spring Boot Starter
 
 This module provides Spring auto-configuration on top of the `axon-tracing` module
+
+## Axon Bill of Materials
+
+Next to the main framework modules and the extensions, Axon also has a [Bill of Materials](https://en.wikipedia.org/wiki/Software_bill_of_materials), or BOM. 
+The BOM is provided to ensure the use of compatible framework and extension dependencies inside an Axon application.
+As such, it is the recommended approach towards defining the overall Axon version used inside an application.
+
+| Module | Artifact Id | Group Id | Maven Central | GitHub |
+| :--- | :--- | :--- | :--- | :---: |
+| [Axon BOM](#axon-bill-of-materials) | axon-bom | org.axonframework | [available](https://search.maven.org/search?q=a:axon-bom) | [available](https://github.com/AxonFramework/axon-bom) |
+
+For using the BOM, you would add the `axon-bom` dependency to your dependency management system:
+
+{% tabs %}
+{% tab title="Maven" %}
+```xml
+
+...
+<dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.axonframework</groupId>
+            <artifactId>axon-bom</artifactId>
+            <version>${version.axon}</version>
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+
+        ...
+
+    </dependencies>
+</dependencyManagement>
+...
+```
+
+{% endtab %}
+
+{% tab title="Gradle" %}
+
+For usage with Gradle, apply the dependency-management-plugin plugin like so:    
```

Comment:
```suggestion
For usage with Gradle, apply the dependency-management-plugin like so:    
```
I don't think you'd need plugin again, since it's already in the name.

Reply:
Completely agree with you there.